feature/experimental/p4rt/wbb/wbb.go

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2022 Cisco Systems, Inc. and its affiliates
+ * All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package wbb
+
+import (
+	"github.com/golang/glog"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
+)
+
+// Some hardcoding to simplify things
+var (
+	WbbTableMap = map[string]uint32{
+		"acl_wbb_ingress_table": 33554691,
+	}
+	WbbActionsMap = map[string]uint32{
+		"acl_wbb_ingress_copy": 16777479,
+		"acl_wbb_ingress_trap": 16777480,
+	}
+	WbbMatchMap = map[string]uint32{
+		"is_ipv4":       1,
+		"is_ipv6":       2,
+		"ether_type":    3,
+		"ttl":           4,
+		"outer_vlan_id": 5,
+	}
+)
+
+type AclWbbIngressTableEntryInfo struct {
+	Type            p4_v1.Update_Type
+	IsIpv4          uint8
+	IsIpv6          uint8
+	EtherType       uint16
+	EtherTypeMask   uint16
+	Ttl             uint8
+	TtlMask         uint8
+	OuterVlanId     uint16 // lower 12 bits
+	OuterVlanIdMask uint16 // lower 12 bits
```

Comment:
OuterVlanIDMask instead of OuterVlanIdMask

Reply:
Changes are done and pushed.